LICENSE

```diff
@@ -26,3 +26,38 @@ CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
 LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY
 OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH
 DAMAGE.
+
+--------------------------------------------------------------------------------
+
+Patches by Sergey B Kirpichev <skirpichev@gmail.com> merged after January 1,
+2016 are licensed as:
+
+Copyright (c) 2006-2015 SymPy Development Team,
+              2015-2016 Sergey B Kirpichev
```

Comment:
Given that there are patches from 2017 in diofant, this really should have been,
```
Copyright (c) 2006-2017 SymPy Development Team,
              2013-2017 Sergey B Kirpichev
```
to comply with SymPy's license